doc/DEPLOYMENT.md

```diff
@@ -851,7 +855,7 @@ Description=STF s3 storage
 After=docker.service
 Requires=docker.service
 
-[Service]
+[Service]provider
```

Comment:
Why we need a `provider` suffix there?

Reply:
I think that was a typo on my part--computer was crashing (Mac's kernel_task taking up 18000% CPU...)

Reply:
ok fixed. How/where do I sign off on this?